graphql2/graphqlapp/escalationpolicy.go

```diff
@@ -19,12 +19,30 @@ import (
 
 type EscalationPolicy App
 type EscalationPolicyStep App
+type CreateEscalationPolicyStepInput App
 
 func (a *App) EscalationPolicy() graphql2.EscalationPolicyResolver { return (*EscalationPolicy)(a) }
 func (a *App) EscalationPolicyStep() graphql2.EscalationPolicyStepResolver {
 	return (*EscalationPolicyStep)(a)
 }
 
+func (a *App) CreateEscalationPolicyStepInput() graphql2.CreateEscalationPolicyStepInputResolver {
+	return (*CreateEscalationPolicyStepInput)(a)
+}
+
+func (a *CreateEscalationPolicyStepInput) Actions(ctx context.Context, input *graphql2.CreateEscalationPolicyStepInput, actions []graphql2.DestinationInput) error {
+	tgts := make([]assignment.RawTarget, len(actions))
+	var err error
+	for i, action := range actions {
+		tgts[i], err = CompatDestToTarget(action)
```

Comment:
Let's call ValidateDestination here before mapping to target, that will give us proper error code responses if any of the destinations are invalid.

Make sure the `fieldName` includes the index